src/config.rs

```diff
@@ -574,6 +574,7 @@ create_config! {
     // function decl?
     // 2. Currently options `Tall` and `Vertical` produce the same output.
     where_density: Density, Density::Vertical, false, "Density of a where clause";
+    where_single_line: bool, false, false, "To force single line where layout";
```

Comment:
Both the name of the option (where single line do what?) and the "documentation" (where what is layout?) are extremely confusing. `Configurations.md` does not help; as a user I have no idea what this does.